recipes/slamdunk/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set version = "0.3.1" %}
+{% set sha256 = "2cc5d561ef2c3014595b646858ae2fa76c3d3972a2b624dd5d01eb82ac239c28" %}
+
+package:
+  name: slamdunk
+  version: {{ version }}
+
+source:
+  sha256: '{{sha256}}'
+  url: "https://github.com/t-neumann/slamdunk/archive/v0.3.1.tar.gz"
+
+build:
+  entry_points:
+    - slamdunk=slamdunk.slamdunk:run
+    - alleyoop=slamdunk.alleyoop:run
+    - splash=slamdunk.splash:run
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - wget
+   
+  host:
+    - python=2.7
```

Comment:
please skip python 3 build in the build/skip section.

Reply:
This should suffice right?

```
build:
  skip: True  # [not py27]
```

Reply:
Whenever I skip python3 build, it won't build my package anymore.

Neither
```
skip: True  # [not py27]
```

nor

```
skip: True  # [py3k]
```

works. So I'm leaving this for now.